src/braket/circuits/circuit.py

```diff
@@ -1270,13 +1270,15 @@ def _generate_frame_wf_defcal_declarations(
                     gate, qubits = key
 
                     # ignoring parametric gates
-                    if hasattr(gate, "angle") and isinstance(gate.angle, FreeParameter):
+                    if hasattr(gate, "parameters") and any(
+                        isinstance(parameter, FreeParameter) for parameter in gate.parameters
+                    ):
                         continue
```

Comment:
I think it would be confusing for customers if the parametric gates are silently ignored.